libs/components/datetime/src/lib/modules/datepicker/calendar/daypicker-cell.component.spec.ts

```diff
@@ -15,43 +15,67 @@ import { SkyDatepickerCalendarService } from './datepicker-calendar.service';
 import { SkyDayPickerCellComponent } from './daypicker-cell.component';
 import { SkyDayPickerContext } from './daypicker-context';
 
-function getDaypickerCell(
+class MockSkyCalendarInnerComponent {
```

Comment:
👇 These specs needed a makeover; I removed a handful of duplicate assertions, and some irrelevant tests that were spying on internal workings. The tests now ask questions about the status of the DOM, instead of verifying if methods were called by internal services.

Largely, the same questions are being answered in this refactor; since we're now using signals for the inputs, I needed to make adjustments to how the input values were being set.